Source/WTF/wtf/PlatformHave.h

```diff
@@ -1459,3 +1459,7 @@
     || (PLATFORM(IOS) || PLATFORM(WATCHOS)))
 #define HAVE_AVAUDIOSESSION_SMARTROUTING 1
 #endif
+
+#if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 140000)
+#define HAVE_NSCOLOR_FILL_COLOR_HIERARCHY 1
```

Comment:
I would call this `HAVE_NSCOLOR_TERTIARY_SYSTEM_FILL_COLOR`.

Reply:
Well, why not allow future users of other fill colors to be able to reuse this definition? 😉 
(All fill colors were added together in rdar://104764768 (Add "fill" color hierarchy to NSColor).)

Reply:
That's a good point! We can leave as-is.